app/mailers/app_mailer.rb

```diff
@@ -13,4 +13,9 @@ def send_forgot_password(user)
     @user = user
     mail to: user.email, from: "info@myflix.com", subject: "Please reset your password"
   end
+
+  def send_invitation_email(invitation)
+    @invitation = invitation
+    mail to: invitation.recipient_email, from: "info@myflix.com", subject: "Invitation to join MyFlix!"
```

Comment:
You could use `default from: " ... " ` as you use `before_action` in controller. Check Rails docs about it.